### PR TITLE
Update demographics for india

### PR DESCRIPTION
**Pull request by jdebacker**
This PR add updated demographics based on the 2011 India census.

### CODE REVIEW

File: ogindia/demographics.py [hunk 1]
```diff
@@ -41,21 +40,20 @@ def get_fert(totpers, min_yr, max_yr, graph=False):
...
+    # Get current population data (2011) for weighting
     cur_path = os.path.split(os.path.abspath(__file__))[0]
+    # Get current population data (2011) for weighting
     pop_file = utils.read_file(cur_path,
-                               "data/demographic/pop_data.csv")
-    pop_data = pd.read_csv(pop_file, sep=',', thousands=',')
-    pop_data_samp = pop_data[(pop_data['Age'] >= min_yr - 1) &
-                             (pop_data['Age'] <= max_yr - 1)]
-    curr_pop = np.array(pop_data_samp['2013'], dtype='f')
+                               "data/demographic/india_pop_data.csv")
```
Comment (jdebacker) [inline ogindia/demographics.py:47]: What is going on here?

Comment (pbpramanik) [reply ogindia/demographics.py:47]: Some more changes are required in demographics to give it a more realistic picture.

**Comment by rickecon** [comment]
@jdebacker. Is this ready to be merged? It looks good to me.

**Comment by jdebacker** [comment]
@rickecon - yes, please merge.

**Pull request merged**